.gitignore

```diff
@@ -4,3 +4,7 @@ _build/*
 __pycache__
 .netlify
 node_modules
+
+# ignore common virtual environment names
```

Comment:
Remove this because we don't know what the user likes to call their directories - but in the README, let's prompt them to add whatever they chose to `.gitignore`. If they get this step wrong, it won't impact on their ability to work with the tool and we would pick up at review if they'd added something they shouldn't have (I hope!)